commands/user_e2e_test.go

```diff
@@ -136,3 +136,36 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestUpdateUserEmailCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("admin and local can change basic user email", func(c client.Client) {
+		newEmail := "basicuser2@fakedomain.com"
+		err := updateUserEmailCmdF(c, &cobra.Command{}, []string{s.th.BasicUser2.Email, newEmail})
+		s.th.BasicUser2.Email = newEmail
```

Comment:
We should update the user's email in the database instead of just in the struct:

```suggestion
		s.th.BasicUser2.Email = newEmail
		_, err := s.th.App.UpdateUser(user, false)
		s.Require().Nil(err)
```

this update should happen after checking that the email value has changed in the database